spec/features/etd_creation_spec.rb

```diff
@@ -216,40 +216,32 @@
     expect(apo_element.first('a')[:href]).to end_with('druid:bx911tp9024') # this is hardcoded in hydra_etd app
     status_element = find_table_cell_following(header_text: 'Status')
     expect(status_element).to have_text('v1 Registered')
-    if Settings.folio.enabled
-      reload_page_until_timeout! do
+
+    Timeout.timeout(Settings.timeouts.workflow) do
+      workflow_done = false
```

Comment:
this is clunky, but apparently "break" only gets out of the "within" and we needed to get out of both the "within" and the "loop" statement.